Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStoreCollection.h

```diff
@@ -61,9 +53,10 @@ class ThreadSafeImageBufferSetFlusher;
 enum class BufferInSetType : uint8_t;
 enum class SwapBuffersDisplayRequirement : uint8_t;
 
-class RemoteLayerBackingStoreCollection : public CanMakeWeakPtr<RemoteLayerBackingStoreCollection> {
+class RemoteLayerBackingStoreCollection final : public CanMakeWeakPtr<RemoteLayerBackingStoreCollection>, public CanMakeCheckedPtr<RemoteLayerBackingStoreCollection> {
```

Comment:
The ownership model is pretty clear here, `RemoteLayerBackingStoreCollection` is owned by `RemoteLayerTreeContext` which itself is ref-counted. I think we should forward the ref-counting.

Reply:
Hm... I swear I saw code which clears the variable but I can't find it anymore. Maybe I was getting confused with some other class. There are quite few of these RemoteLayerBackingStore* classes.